load-tests/file_server/src/main.bal

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/lang.runtime;
+import ballerina/time;
+import ballerina/udp;
+
+int errorCount = 0;
+int sentCount = 0;
+int receivedCount = 0;
+int deletedCount = 0;
+time:Utc startedTime = time:utcNow();
+time:Utc endedTime = time:utcNow();
+boolean finished = false;
+const int PORT = 8080;
+const int TERMINAL = 14;
+service /udp on new http:Listener(9100) {
+
+    resource function get publish() returns boolean {
+        error? result = startListener();
+        if result is error {
+            return false;
+        }
+        errorCount = 0;
+        sentCount = 0;
+        startedTime = time:utcNow();
+        endedTime = time:utcNow();
+        finished = false;
+        _ = start publishMessages();
+        return true;
+    }
+
+    resource function get getResults() returns boolean|map<string> {
+        if finished {
+            return {
+                errorCount: errorCount.toString(),
+                time: time:utcDiffSeconds(endedTime, startedTime).toString(),
+                sentCount: sentCount.toString(),
+                receivedCount: receivedCount.toString()
+            };
+        }
+        return false;
+    }
+}
+
+public function publishMessages() returns error? {
+    udp:ConnectClient socketClient = check new ("localhost", PORT, timeout = 1);
+    startedTime = time:utcNow();
+    int endingTimeInSecs = startedTime[0] + 200;
+    byte sequenceNo = 0;
+    while time:utcNow()[0] <= endingTimeInSecs {
```

Comment:
We could refactor this as follows:
```ballerina
byte sequenceNo = 0;
time:Utc expiryTime = time:utcAddSeconds(startedTime, 200);
while time:utcDiffSeconds(expiryTime, time:utcNow()) > 0D {
  // logic here
}
```

Reply:
Fixed with [1a4259f](https://github.com/ballerina-platform/module-ballerina-udp/pull/394/commits/1a4259fbf6aff42f6aa32fc4e64db7dd96e5f304)